device-types/Eaton/Tripp-Lite-PDUH20DV.yaml

```diff
@@ -0,0 +1,40 @@
+---
+manufacturer: Eaton
+model: Tripp Lite PDUH20DV
+slug: eaton-tripp-lite-pduh20dv
+u_height: 1
+is_full_depth: false
+front_image: true
+rear_image: true
```

Comment:
1. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. This appears to be listed in the datasheet (3.75 lbs or 1.70 kg)